server/package.json

```diff
@@ -59,7 +59,7 @@
     "eslint-plugin-import": "2.22.0",
     "eslint-plugin-jest": "23.20.0",
     "eslint-plugin-prettier": "3.4.0",
-    "jest": "25.3.0",
+    "jest": "28.0.2",
```

Comment:
Why is it necessary to update `jest` version?

Reply:
This is result of audit fix